scripts/configure_pyproject_toml.py

```diff
@@ -60,7 +86,7 @@ def parse_args():
         "cmake~=3.24.0",
         "ninja; platform_system!='Windows'",
         "setuptools>=42",
-        "toml",
+        "tomlkit",
```

Comment:
If `tomlkit` is not found, do we want to ship with `toml`?

Reply:
We don't want to ship our packages with toml anymore. we should be able to remove `toml` from the list of `toml_libs` when PL drops support for `toml` in the future too. For now, we only rely on `tomlkit.load` when installing Lightning PyPI packages and it's available in every platform.